### PR TITLE
Visual Editor: Disallow to choose already selected fields

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves [#249](https://github.com/VolkovLabs/volkovlabs-echarts-panel/issues/249)

### CODE REVIEW

File: src/components/DatasetEditor/DatasetEditor.test.tsx [hunk 1]
```diff
@@ -159,6 +159,30 @@ describe('Dataset Editor', () => {
...
+        onChange,
+      })
+    );
+
+    const newItemNameInput = selectors.newItemName();
+
+    /**
+     * Simulate select option doesn't exist
+     */
+
```
Comment (asimonok) [inline src/components/DatasetEditor/DatasetEditor.test.tsx:180]: please remove empty line

File: src/components/DatasetEditor/DatasetEditor.test.tsx [hunk 2]
```diff
@@ -159,6 +159,30 @@ describe('Dataset Editor', () => {
...
+     * Render
+     */
+    render(
+      getComponent({
+        value,
+        onChange,
+      })
+    );
+
+    const newItemNameInput = selectors.newItemName();
```
Comment (asimonok) [inline src/components/DatasetEditor/DatasetEditor.test.tsx:175]: variable is not needed here because selector is used once

**Review by asimonok — CHANGES_REQUESTED** [review]

File: src/components/DatasetEditor/DatasetEditor.test.tsx (hunk 1 above)
Comment (vitPinchuk) [reply src/components/DatasetEditor/DatasetEditor.test.tsx]: removed

File: src/components/DatasetEditor/DatasetEditor.test.tsx (hunk 2 above)
Comment (vitPinchuk) [reply src/components/DatasetEditor/DatasetEditor.test.tsx]: fixed

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**